File: src/main/java/com/beat/global/external/s3/exception/FileSuccessCode.java
package com.beat.global.external.s3.exception;

import com.beat.global.common.exception.base.BaseSuccessCode;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public enum FileSuccessCode implements BaseSuccessCode {
    PERFORMANCE_MAKER_PRESIGNED_URL_ISSUED(200, "공연 메이커를 위한 Presigned URL 발급 성공");
    private final int status;
    private final String message;
}

File: apps/api/src/main/kotlin/com/beat/apps/api/file/api/response/FileSuccessCode.kt
package com.beat.apps.api.file.api.response

import com.beat.apps.api.response.SuccessCode

enum class FileSuccessCode(
    override val status: Int,
    override val message: String,
) : SuccessCode {
    PERFORMANCE_MAKER_PRESIGNED_URL_ISSUED(200, "공연 메이커를 위한 Presigned URL 발급 성공")
}
